src/App/Tracker/Controller/Upload/Ajax/Put.php

```diff
@@ -42,20 +42,33 @@ protected function prepareResponse()
 			$destDir    = $application->getProject()->project_id;
 			$fullPath   = $host . '/' . $application->get('system.upload_dir') . '/' . $destDir . '/' . $destName;
 
+			$isImage = (false === strpos($file->getMimetype(), 'image')) ? false : true;
```

Comment:
Well, not really.  In that case, if 'image' is at the first character position `strpos()` will return 0 and if it isn't in the string it'll return false.  PHP treats the two values as the same, so you need the explicit `!== false` check (and it can't be `!=` because thanks to PHP's weak type system you'd end up with a 0 != false comparison which doesn't evaluate right).  Gotta love PHP sometimes.